httpobsdashboard/conf/deviations.json

```diff
@@ -180,5 +180,42 @@
     "tlsobs": {
       "pass": true
     }
+  },
+  "HTTP Only": {
+    "httpobs": {
+      "tests": {
+        "strict-transport-security": {
+          "pass": null,
+          "score_description": "Allow HTTP without an HTTPS redirect.",
+          "score_modifier": 0
+        },
+        "redirection-missing": {
+          "pass": null,
+          "score_description": "Allow HTTP without an HTTPS redirect.",
+          "score_modifier": 0
+        }
+      }
+    }
+  },
+  "Package Repository": {
```

Comment:
Looking at this, can we instead use `Static Community Site`, rename it to `Static Site`, and update the descriptions a bit?